src/docs/truffle/how-to/migrate-to-hardhat.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Migrate to Hardhat
+layout: docs.hbs
+---
+
+# Migrate to Hardhat
+
+Like Truffle, Hardhat is a development environment for Ethereum software. It consists of different components for
+editing, compiling, debugging and deploying smart contracts and dapps.
+
+This topic provides some general steps you need to perform to migrate an existing Truffle project to Hardhat.
+
+## Install Hardhat
+
+Install Hardhat using the following command:
+
+```bash
+npm install --save-dev hardhat
+```
+
+## Update the folder structure
+
+Update the Truffle folder structure as follows:
+
+* Rename the `migrations` directory to `scripts`
+* Rename `truffle-config.js` to `hardhat.config.js`
+
+The standard folder structure for Truffle  and Hardhat is as follows:
+
+=== "Truffle folder structure"
+
+    ```
+    truffle
+    ├── truffle-config.js       
+    ├── contracts               // source files for your contracts 
+    ├── migrations              // location for your scripts, for example to deploy to a chain
+    ├── test                    // contract tests
+    ```
+
+=== "Hardhat folder structure"
+
+    ```
+    hardhat
+    ├── hardhat.config.js       // hardhat network config
+    ├── contracts               // source files for your contracts 
+    ├── scripts                 // location for your scripts, for example to deploy to a chain
+    ├── test                    // contract tests
+    ```
+
+## Compile, test, and deploy contracts
+
+The process to compile, test, and deploy your contracts on Hardhat is:
+
+1. Install dependencies listed in your project's `package.json` file:
+
+    ```bash
+    npm i
+    ```
+
+2. Compile the contracts in your Hardhat project:
+
+    ```bash
+    npx hardhat compile
+    ```
+
+3. Test your contract:
+
+    ```bash
+    npx hardhat test
+    ```
+
+4. Test a contract deployment:
```

Comment:
Suggest "Hardhat uses ethers.js so you will likely need to swap web3 for ethers and you can then deploy you contract. Please refer to hardhat [docs](https://hardhat.org/hardhat-runner/docs/guides/deploying) for more information on deploying contracts"